docs/tools/buildkit.md

```diff
@@ -135,6 +108,84 @@ If you want to ensure that the buildkit CLI tools are always available, then:
     Each time you open a new terminal while working on Civi development, you would need to re-run the `export` command.
 
 
+### Configuring `amp` {:#amp-config}
+
+Buildkit provides a tool called `amp` which [civibuild](/tools/civibuild.md) uses when it needs to set up a new site. Before you can use `civibuild`, need to configure `amp` by telling it a bit about your system (e.g. what webserver you're using). 
+
+#### Interactive config
+
+```
+$ amp config
+```
+
+!!! tip "tips"
+    * Run this as a non-`root` user who has `sudo` permission. This will ensure that new files are owned by a regular user, and (if necessary) it enables `civibuild` to restart your webserver and edit `/etc/hosts`.
+    * Pay close attention to any instructions given in the output of this command.  They may involve adding a line to your Apache or nginx configuration file.
+    * To check which version of apache you have, run `apachectl -v`
+
+#### Testing amp's configuration {:#amp-test} 
+
+Test that `amp` is correctly configured.
+
+```
+$ amp test
+```
+
+!!! failure "Troubleshooting errors from `amp test`"
+    
+    * Try manually adding settings to your sebserver, as described below.
+    * Re-run `amp config`.
+
+#### Manually adding settings to your webserver {:#amp-webserver}
+
+During one of the steps in the `amp config` process, `amp` will *attempt to* alter the system-wide settings for your local webserver (apache or nginx). On some platforms, `amp` is not able to perform the necessary configuration and you must do it manually by following these steps:
```

Comment:
This seems like a misunderstanding -- `amp` makes *no attempt* to modify the system-wide httpd configuration. Rather, it *tells the user to do it* (and tries to give some pointers in the right direction). Of course, half the users ignore those instructions, so it's a recurring pain-point. 

Random:
 * Guess: there's confusion because the vagrant and docker builds do that part automatically. 
 * But as far as `amp config` and manual setup are concerned, these aren't automated.
 * It would be patch-welcome for `amp config` to put up a better prompt.
 * It would be patch-welcome for `amp config` to try to apply the needful.
 * It would be patch-welcome for `civi-download-tools --full` to try to apply the needful.